src/OrchardCore.Modules/OrchardCore.Tenants/wwwroot/Scripts/tenant.js

```diff
@@ -17,5 +17,9 @@ function toggleConnectionStringAndPrefix() {
         $(this).data("table-prefix") === true
             ? $(".tablePrefix").show()
             : $(".tablePrefix").hide();
+
+        var dataProvider = $(this).text().replace(/\s/g, '').toLowerCase();
+        var connectionStringHint = $("#connectionStringHint").data(dataProvider + "-connectionstring");
```

Comment:
Wouldn't it make the code simpler if the text was in the a data attribute of each `<option>` tag?